### PR TITLE
Create support playbbook

### PR DESCRIPTION
**Pull request by darokel**
### Context

Any support request outside of the support console, we have to action manually. We need a playbook so it's easier to perform these requests.

### Changes proposed in this pull request

Create a playbook

### Guidance to review

Check for typos etc

### CODE REVIEW

File: docs/support_playbook.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Removing an organisation
+
+To remove an organisation, you can `discard` it:
+
+```ruby
+RecruitmentCycle.current.providers.find_by(provider_code: "1JJ").discard
+```
+
+If the org has running courses, you will get a validation error. In this scenario you just need to make sure there aren't any users attached to it.
```
Comment (defong) [inline docs/support_playbook.md:12]:
```suggestion
If the organisation has running courses, you will get a validation error. In this scenario you just need to make sure there aren't any users attached to it.
```

File: docs/support_playbook.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Republishing a withdrawn course
+
+To republish a course which has been withdrawn:
+
+```ruby
+# Find the course by code or urn
+course = RecruitmentCycle.current.providers.find_by(urn: "138459").courses.find_by(course_code: "3C2F")
```
Comment (defong) [inline docs/support_playbook.md:20]: probably best to user `provider_code` or `id` as `urn` has no unique constraint

Comment (darokel) [reply docs/support_playbook.md]: done

File: docs/support_playbook.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+## Changing course to a different type (eg to salaried)
+
+Change `program_type` to the preferred value on the course record.
+
+## Changing provider to a different type (eg scitt to sd)
+
+Change `provider_type` to the preferred value on the provider record.
+
+
+## Changing accredited body of courses
```
Comment (defong) [inline docs/support_playbook.md:38]:
```suggestion

## Changing accredited body of courses
```

**Review by defong — APPROVED** [review]

**Pull request merged**